src/main/java/duke/StringConstants.java

```diff
@@ -0,0 +1,24 @@
+package duke;
```

Comment:
Should these constants be shifted and declared in the UI class? I'm not sure if creating a class just for these message constants is necessary.